driver/pace/driver/driver.py

```diff
@@ -598,11 +599,16 @@ def _critical_path_step_all(
     def step_all(self):
         logger.info("integrating driver forward in time")
         with self.performance_collector.total_timer.clock("total"):
+            self.profiler.enable()
```

Comment:
It doesn't actually make sense to have this for dace orchestration, do we want to support this for other backends?

Reply:
I would keep it here